repositories.yaml

```diff
@@ -10,7 +10,7 @@ libraries:
     description: Modern cross-platform telemetry (core library)
     notification_emails:
       - dthorn@mozilla.com
-      - mdroettboom@mozilla.com
+      - aplacitelli@mozilla.com
```

Comment:
We can also future proof ourselves by adding glean-team@mozilla.com in addition to me, here.